controller_interface/src/controller_interface.cpp

```diff
@@ -25,30 +26,105 @@ namespace controller_interface
 return_type
 ControllerInterface::init(const std::string & controller_name)
```

Comment:
This is an opportunity to remove this unused parameter

```suggestion
 ControllerInterface::init()
```

Reply:
But it is not unused, we use to initialize the node.
Also I think it's interesting that controllers are aware of the name they were instantiated with.